action.yml

```
@@ -1,13 +1,19 @@
 name: "release extension"
 description: "Generate a release for a precompiled gh extension"
 inputs:
-  gpg_fingerprint:
-    description: "GPG fingerprint to use for signing releases"
   build_script_override:
     description: "Path to custom build script for producing binaries to upload"
+  draft_release:
+    description: "Create a draft release"
+  github_token:
+    description: "GitHub token to use, defaults to `github.token` if unspecified"
   go_version:
     description: "The Go version to use for compiling (supports semver spec and ranges)"
     default: "1.18"
+  gpg_fingerprint:
+    description: "GPG fingerprint to use for signing releases"
+  release_tag:
+    description: "Existing tag that the release should be created from, defaults to `github.ref` if unspecified"
```

Super small wording nit.

```suggestion
    description: "Tag that the release should be created from, defaults to `github.ref` if unspecified"
```

---

Addressed in [289fe4e](https://github.com/cli/gh-extension-precompile/pull/37/commits/289fe4e3d2e4cc78a1f99c746486254badad8d90)